helm-chart/templates/cvat_kvrocks/deployment.yml

```diff
@@ -0,0 +1,90 @@
+{{- if .Values.cvat.kvrocks.enabled }}
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: {{ .Release.Name }}-kvrocks
+  namespace: {{ .Release.Namespace }}
+  labels:
+    {{- include "cvat.labels" . | nindent 4 }}
+    app: cvat-app
+    tier: kvrocks
+spec:
+  replicas: {{ .Values.cvat.kvrocks.replicas }}
+  strategy:
+    type: Recreate
+  selector:
+    matchLabels:
+      {{- include "cvat.labels" . | nindent 6 }}
+      app: cvat-app
+      tier: kvrocks
+  template:
+    metadata:
+      labels:
+        {{- include "cvat.labels" . | nindent 8 }}
+        app: cvat-app
+        tier: kvrocks
+        {{- with .Values.cvat.kvrocks.labels }}
+        {{- toYaml . | nindent 8 }}
+        {{- end }}
+        {{- with .Values.cvat.kvrocks.annotations }}
+      annotations:
+        {{- toYaml . | nindent 8 }}
+        {{- end }}
+    spec:
+      containers:
+        - name: cvat-kvrocks-app-container
+          image: {{ .Values.cvat.kvrocks.image }}:{{ .Values.cvat.kvrocks.tag }}
+          imagePullPolicy: {{ .Values.cvat.kvrocks.imagePullPolicy }}
+          args:
+            - --dir
+            - /var/lib/kvrocks/data
+            - --requirepass
+            - "$(CVAT_REDIS_ONDISK_PASSWORD)"
```

Comment:
When does this variable get expanded? It is Kvrocks that does it?

Reply:
this var is expanded by k8s: https://kubernetes.io/docs/tasks/inject-data-application/define-command-argument-container/#use-environment-variables-to-define-arguments

Reply:
I see. Good to know.